Don't use VMConfig.VCPUs

# Description of problem

failed to build cc-runtime

# github.com/clearcontainers/runtime
./config.go:331:19: config.VMConfig.VCPUs undefined (type virtcontainers.Resources has no field or method VCPUs)
make: *** [Makefile:440: cc-runtime] Error 2

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -50,7 +50,7 @@
     "pkg/uuid",
     "pkg/vcMock"
   ]
-  revision = "1dce29db26331bb48be149ef6a1254483128aa89"
+  revision = "4fb2dacea4ab8b15807fa982e695487c9a20dc46"
 
 [[projects]]
   name = "github.com/davecgh/go-spew"
@@ -249,6 +249,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "e36a22c9fa6bbb0ad20dd93f678582fc06253f5c82c8d23654ac5c50e53f7d34"
+  inputs-digest = "eac859455568d17a23cb3dbb7ebff8ce6706d1e0d7ebb8c34b962be38842a51b"
   solver-name = "gps-cdcl"
   solver-version = 1

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -24,7 +24,7 @@
 
 [[constraint]]
   name = "github.com/containers/virtcontainers"
-  revision = "1dce29db26331bb48be149ef6a1254483128aa89"
+  revision = "4fb2dacea4ab8b15807fa982e695487c9a20dc46"
 
 [prune]
   non-go = true

--- a/config.go
+++ b/config.go
@@ -328,7 +328,6 @@ func updateRuntimeConfig(configPath string, tomlConf tomlConfig, config *oci.Run
 				return fmt.Errorf("%v: %v", configPath, err)
 			}
 
-			config.VMConfig.VCPUs = uint(hConfig.DefaultVCPUs)
 			config.VMConfig.Memory = uint(hConfig.DefaultMemSz)
 
 			config.HypervisorConfig = hConfig

--- a/config_test.go
+++ b/config_test.go
@@ -166,7 +166,6 @@ func createAllRuntimeConfigFiles(dir, hypervisor string) (config testRuntimeConf
 		ShimConfig: shimConfig,
 
 		VMConfig: vc.Resources{
-			VCPUs:  uint(defaultVCPUCount),
 			Memory: uint(defaultMemSize),
 		},
 	}
@@ -1026,7 +1025,6 @@ func TestUpdateRuntimeConfigurationVMConfig(t *testing.T) {
 
 	config := oci.RuntimeConfig{}
 	expectedVMConfig := vc.Resources{
-		VCPUs:  vcpus,
 		Memory: mem,
 	}
 

